Code/GraphMol/Wrap/rough_test.py

```diff
@@ -3317,7 +3313,12 @@ def testCrambin(self):
   def testAtomBondProps(self):
     m = Chem.MolFromSmiles('c1ccccc1')
     for atom in m.GetAtoms():
-      self.assertEquals(atom.GetPropsAsDict(), {'_CIPRank': 0})
+      d = atom.GetPropsAsDict()
+      self.assertEquals(set(d.keys()), set(['_CIPRank', '__computedProps']))
```

Comment:
I'm pretty unclear why this regression tests changed, the semantics of GetPropNames is the same as before, I don't know why the earlier one didn't behave this way.